meetup/README.md

```diff
@@ -0,0 +1,11 @@
+# meetup rsvps
+This tool prints out a list of full names of those who RSVPed yes for the given meetup. The names are those as shown in the Chipy user profile section where the full name is asked.
+
+# requirements
+- Requests
+- Beautiful Soup
```

Comment:
which version? maybe it'll be useful to have a `requirements.txt`

Reply:
Any version works it's very basic functionality